ApplicationUser.cs

```diff
@@ -0,0 +1,7 @@
+using Microsoft.AspNetCore.Identity;
```

Comment:
I think this file belong to the next cr?

Also, avoid using AspNet Identity service specially the DB side. Only use the API Level concepts.